examples/kubernetes/docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+services:
+  optimum-habana:
+    build:
+      args:
+        http_proxy: ${http_proxy}
+        https_proxy: ${https_proxy}
+        no_proxy: ""
+        BASE_IMAGE_NAME: ${BASE_IMAGE_NAME:-vault.habana.ai/gaudi-docker/1.15.1/ubuntu22.04/habanalabs/pytorch-installer-2.2.0}
+        BASE_IMAGE_TAG: ${BASE_IMAGE_TAG:-latest}
+        GAUDI_SW_VER: 1.15.1
+        OPTIMUM_HABANA_VER: 1.11.1
+        REGISTRY: ${REGISTRY}
+        REPO: ${REPO}
+      context: .
+      labels:
+        org.opencontainers.base.name: "vault.habana.ai/gaudi-docker/1.15.1/ubuntu22.04/habanalabs/pytorch-installer-2.2.0:latest"
```

Comment:
I'd replace `1.15.1` with `${GAUDI_SW_VER:-1.15.1}` to stay consistent.

Reply:
Thanks, I updated it to use `BASE_IMAGE_NAME` and `BASE_IMAGE_TAG`